marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -214,6 +219,10 @@ class BrowsePage {
     return cy.get('[data-cy=table-view]').click();
   }
 
+  getSideBarCollapseIcon() {
+    return cy.get('#sidebar-collapse-icon');
```

Comment:
@timur-isangulov is this method being used ?

Reply:
This function might be useful for the future tests. Srikanth made a commit with the test, probably shouldn't squash.